client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -201,6 +201,32 @@ Undocumented.prototype.jetpackAuthorizeSSONonce = function( siteId, ssoNonce, fn
 	return this.wpcom.req.post( { path: endpointUrl }, params, fn );
 };
 
+/**
+ * Gets the current status of a full sync for a Jetpack site.
+ *
+ * @param {int|string} siteId The site ID
+ * @param {Function} fn The callback function
+ * @api public
+ */
+Undocumented.prototype.getJetpackSyncStatus = function( siteId, fn ) {
+	debug( '/sites/:site_id:/sync/status query' );
+	const endpointPath = '/sites/' + siteId + '/sync/status';
+	return this.wpcom.req.get( { path: endpointPath }, fn );
+};
+
+/**
+ * Schedules a full sync for a Jetpack site.
+ *
+ * @param {int|string} siteId The site ID
+ * @param {Function} fn The callback function
+ * @api public
+ */
+Undocumented.prototype.scheduleJetpackFullysync = function( siteId, fn ) {
+	debug( '/sites/:site_id:/sync query' );
+	const endpointPath = '/sites/' + siteId + '/sync';
+	return this.wpcom.req.post( { path: endpointPath }, fn );
```

Comment:
Have you tested this? I don't think it will work as expected for you. 
I think it should be more like 
`return this.wpcom.req.post( { path: endpointPath }, {},  fn );` Take a look at 
https://github.com/Automattic/wpcom.js/blob/master/lib/util/request.js#L44